org.bridgedb.uri.api/test/org.bridgedb.statistics/DataSetInfoTest.java

```diff
@@ -0,0 +1,42 @@
+package org.bridgedb.statistics;
```

Comment:
Please add a copyright/license info (many BridgeDb `.java` files don't have that yet).

With a format like: https://github.com/cdk/cdk/blob/master/base/core/src/main/java/org/openscience/cdk/AtomRef.java

And the license like: https://github.com/bridgedb/BridgeDb/blob/master/org.bridgedb.bio/src/org/bridgedb/bio/BioDataSource.java

Reply:
Should I merge both of them to create a new copyright/license info and then add it to the file?

Reply:
Just adding an additional patch is fine!

Reply:
done